docs/src/main/paradox/docs/releases/v1.9-release-notes.md

```diff
@@ -258,9 +261,54 @@ It is now possible to query whether the current logged in user has a specific pe
 
 @ref:[More information](../delta/api/user-permissions-api.md)
 
-## Nexus Fusion
+## Nexus Fusion 1.9 Release Notes
+
+The 1.9 release of Nexus Fusion focuses on enhancing user interaction, resource management, and overall application efficiency. This update introduces new features and key improvements across various sections of the application.
+
+### Home Page
+
+- **Global User Interface Improvements:**
+  - Multi-type filter selection in "My Data" section.
+  - Improved header menu behavior while scrolling.
+  - Enhanced uniformity in UI elements.
+
+### Data Explorer
+
+- **Enhancements:**
+  - New filtering and navigation features.
+  - Fullscreen mode for Graph Flow.
+  - Advanced mode beta for complex operations.
```

Comment:
Advanced mode is actually the "data explorer"
you can added build beta version of data explorer instead of "... for complex ops"